src/util.js

```diff
@@ -13,3 +13,32 @@ export function onlyChild(children) {
 export function filterNullChildren(children) {
 	return children && children.filter(i => i !== null);
 }
+
+export const requestAnimationFrame = (() => {
```

Comment:
It might be worth cutting this down to a simpler check that would cover 95% of cases:

```js
export const requestAnimationFrame = window.requestAnimationFrame || setTimeout;
```

Reply:
👍